README.md

```diff
@@ -9,7 +9,7 @@
 Ark gives you tools to back up and restore your Kubernetes cluster resources and persistent volumes. Ark lets you:
```

Comment:
Perhaps we could move some of the wording from the "Copy cluster resources.." change here. Something like `Ark gives you tools to back up and restore your Kubernetes cluster resources and persistent volumes, whether hosted with a cloud provider or on-premises`?